CHANGELOG.md

```diff
@@ -36,6 +36,13 @@ Deprecated modules
 * `Data.List.Relation.Binary.Sublist.Propositional.Disjoint` deprecated in favour of
   `Data.List.Relation.Binary.Sublist.Propositional.Slice`.
 
+Un-deprecated modules
+---------------------
+
+* The modules `Function.Endomorphism.Propositional` and
+  `Function.Endomorphism.Setoid` previously used the old `Function`
+  hierarchy but have now been ported to using the new one.
```

Comment:
Nice addition to the `CHANGELOG` rubric... and definitely preferable to inventing bogus 'alternative' names...?